packages/react-components/src/components/xy-output-component.tsx

```diff
@@ -73,8 +78,12 @@ export class XYOutputComponent extends AbstractTreeOutputComponent<AbstractOutpu
            this.applySelectionZoom();
         }
         this.mouseIsDown = false;
-        if (!this.state.shiftKey) {
-            this.setState({ cursor: 'default'});
+        this.isSelecting = false;
+        if (!this.state.shiftKey && !this.state.ctrlKey) {
+            this.isPanning = false;
+            this.setState({ cursor: 'default' });
+        } else if (this.state.ctrlKey) {
+            this.setState({ cursor: 'grabbing' });
```

Comment:
When you release the mouse then you should neither be isPanning or isSelecting, regardless of keys being pressed (the keys should only affect the cursor).

Reply:
Agreed.